www/index.html

```diff
@@ -694,6 +694,23 @@ <h3 data-i18n="configure-expert-settings-title">Expert settings</h3>
                                         <span data-i18n="configure-expert-bypassappcache"><b>Bypass AppCache</b> (<i>disables offline use of PWA!</i>)</span>
                                     </label>
                                 </div>
+                                <div class="checkbox" id="">
+                                    <label data-i18n-tip="configure-expert-useLibzim-tip" title="Uses the selected version of libzim to access the ZIM contents (ServiceWorker mode only)">
+                                        <input type="checkbox" name="useLibzim" id="useLibzim">
+                                        <span><strong data-i18n="configure-expert-useLibzim">Use Libzim for loading zim</strong>(<i data-i18n="configure-expert-useLibzim-warning">Uses unstable libzim api for reading zim</i>)</span>
+                                    </label>
+                                </div>
+                                <div class="checkbox" id="libzimMode" style="display: none;">
+                                    <label data-i18n-tip="configure-expert-libzimMode-tip" title="Libzim version: (Select which version to use)">
```

Comment:
You shouldn't use a tooltip title that merely says the same thing as the visible text (in essence). So, either remove the tooltip here, or else put something more informative in (e.g. that the setting will refresh the page).